testApi/src/main/scala/africa/absa/testing/scapi/rest/response/action/ExtractJsonResponseAction.scala

```diff
@@ -23,9 +23,12 @@ import africa.absa.testing.scapi.rest.response.action.types.ExtractJsonResponseA
 import africa.absa.testing.scapi.utils.cache.RuntimeCache
 import africa.absa.testing.scapi.utils.validation.ContentValidator
 import africa.absa.testing.scapi.{AssertionException, UndefinedResponseActionTypeException}
+import com.jayway.jsonpath.{Configuration, JsonPath}
 import spray.json._
 
 import scala.util.Try
+import com.jayway.jsonpath.{Option => JsonPathOption}
```

Comment:
```suggestion
import com.jayway.jsonpath.{Configuration, JsonPath, Option => JsonPathOption}
import spray.json._

import scala.util.Try
```

Reply:
Suggestion fixed in commit 50143932f2097dbcc40334baae0c69a39fcffa54.